source/develop/release-management/features/virt/KvmToOvirt.html.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Importing Libvirt KVM VMs to oVirt
+category: feature
+authors: Shahar Havivi
+wiki_category: Feature|v2v
+wiki_title: Features/KvmToOvirt
+wiki_revision_count: 1
+wiki_last_updated: 2017-03-20
+feature_name: 'v2v: Importing Libvirt Kvm Vms to oVirt'
+feature_modules: all
+feature_status: Released in oVirt 4.1
+---
+
+# Importing Libvirt KVM VMs to oVirt
+
+## Summary
+Import KVM based VMs from Libvirt management via the oVirt admin portal.
+There is no need for disk conversion since both oVirt and Libvirt support KVM based VMs.
+Importing VMs with file and block device disks is supported, snapshots are currently not supported. Please note that if you want to import VM with snapshots the current snapshot is the one that will be imported.
+The import process is done via the utility `kvm2ovirt` that comes in VDSM installation. It is located at `/usr/libexec/vdsm/kvm2ovirt`.
+
+## Requirements on VMs
+
+* The VMs that you want to import must be in powered off.
+
+* It is expected that the VirtIO drivers are already installed in the VM. While this is usually already the case for Linux VMs, special care has to be taken for Windows VMs.
```

Comment:
Suggest:
VirtIO drivers should already be installed on the VM. Typically, this is the case for Linux VMs, but not necessarily for Windows VMs.